Classes/Utility/UserUtility.php

```diff
@@ -182,12 +181,8 @@ public static function hashPassword(User &$user, $method)
                 break;
 
             default:
-                if (ExtensionManagementUtility::isLoaded('saltedpasswords')) {
-                    if (SaltedPasswordsUtility::isUsageEnabled('FE')) {
-                        $objInstanceSaltedPw = SaltFactory::getSaltingInstance();
-                        $user->setPassword($objInstanceSaltedPw->getHashedPassword($user->getPassword()));
-                    }
-                }
```

Comment:
Just cause i saw it, while i tried the PR.
The PhpDoc is missing `/** @var PasswordHashInterface $hashInstance */`

Reply:
That is not necessary as `dynamicReturnType` added in https://forge.typo3.org/issues/63553 can display which class is created by makeInstance. 

`DynamicReturnTypePlugin` in PHPStorm.